instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/RouteRefreshTest.kt

```diff
@@ -252,8 +252,8 @@ class RouteRefreshTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.ja
             alternativesMetadata.navigationRoute.id
         )
         assertNotEquals(alternativesMetadataLegacy, alternativesMetadata)
-        assertEquals(267.9, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 1.0)
-        assertEquals(277.0, alternativesMetadata.infoFromStartOfPrimary.duration, 1.0)
+        assertEquals(227.9, alternativesMetadataLegacy.infoFromStartOfPrimary.duration, 1.0)
+        assertEquals(235.0, alternativesMetadata.infoFromStartOfPrimary.duration, 1.0)
```

Comment:
Suppose, it was fixed on NN side `Fixed route parsing. Annotation duration from route is using to get segment's duration.`

Reply:
Right. Now we use durations of segments from annotation object. You can check that new durations are correct using `annotation.duration` array from route's json. Or just delete `annotation.duration` from the route's json